frontend/workflows/projectSelector/src/card.tsx

```diff
@@ -56,7 +56,20 @@ const Card = ({ avatar, children, error, isLoading, summary, title }: CardProps)
           </StyledProgressContainer>
         </CardHeader>
         {expanded && (
-          <CardContent padding={0}>{error ? <Error subject={error} /> : children}</CardContent>
+          <CardContent
+            padding={0}
+            collapsible
+            collapseAction={{
+              title: "See More",
+              dualStateIcon: {
+                true: <KeyboardArrowUpIcon />,
+                false: <KeyboardArrowDownIcon />,
+              },
+            }}
+            maxHeight={400}
```

Comment:
The notes on figma say that at the 6th table row we should collapse the card but not all cards will have tables so I think it should be based on at X height we collapse the card. I think 400px looks good, but can get recommendations from Design